justfile

```diff
@@ -101,15 +101,9 @@ py-build *ARGS:
 py-format:
     #!/usr/bin/env bash
     set -euo pipefail
-    # The order below is important and sadly we need to call black twice. Ruff does not yet
-    # fix line-length (See: https://github.com/astral-sh/ruff/issues/1904).
-    #
-    # 1) Call black, which among others things fixes line-length
-    # 2) Call ruff, which requires line-lengths to be correct
-    # 3) Call black again to cleanup some whitespace issues ruff might introduce
-    black --config rerun_py/pyproject.toml {{py_folders}}
+    # NOTE: the order here matches the one in `crates/re_types_builder/src/format/python.rs`:
     ruff --fix --config rerun_py/pyproject.toml {{py_folders}}
-    black --config rerun_py/pyproject.toml {{py_folders}}
+    ruff format --config rerun_py/pyproject.toml {{py_folders}}
     blackdoc {{py_folders}}
```

Comment:
this seems to imply that there is still need for black? But it was removed from the pixi toml and everywhere else 🤔

Reply:
`blackdoc` formats code in docstrings, which `ruff format` does not yet do: https://github.com/astral-sh/ruff/issues/7146

Reply:
I added a comment about this